### DISCUSSION TITLE
In which liability should I deposit money?

### DISCUSSION THREAD

**Discussion by clouedoc** [firefly-iii/firefly-iii#5458] [Firefly III]
My friend owes me 653.75€.

Here are the liabilities that Firefly created:
![image](https://user-images.githubusercontent.com/13921610/147801378-da3481db-7580-4844-9985-6f11bdd1f4ef.png)

[The documentation](https://docs.firefly-iii.org/firefly-iii/concepts/accounts/#creditor) states that I should deposit money into _the_ liability.
However, Firefly created two liabilities!

In which liability should I transfer money?

Kind regards

**Comment by JC5** [comment]
The second one. The first one is supposed to be hidden, I think you found a bug! 😅
